mautic/mautic
# [UI] Account overview

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR is based [on another PR](https://github.com/mautic/mautic/pull/13735), where I introduce tokens to replace most of the UI variables, aiming to improve consistency in all components. **We need it to be merged before this one becomes available for code review.**

This PR creates a user overview, where we can put toggles for dark theme and high contrast later.
It also indicates where to change profile picture (mautic is based on gravatar).

When someone has a profile pic in their Gravatar profile, it's shown here. The background is automatically created through a blur filter on top of an image copy, so colors combine and get harmonic etc.

![image](https://github.com/mautic/mautic/assets/116097999/f0d58516-c5e2-4952-9452-7a468813061a)

Mobile:
![image](https://github.com/mautic/mautic/assets/116097999/b765058c-8536-4f38-b458-82ad11e68ced)







---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open top navbar > Account > See
